src/carts/Dockerfile

```diff
@@ -1,7 +1,7 @@
 ##### Application Metadata
 ARG APPLICATION_NAME="carts"
 ARG VERSION="1.1.0-SNAPSHOT"
-ARG SODA_VERSION="1.1.0"
+ARG SODA_VERSION="1.1.4"
 ARG clientVersion=19.3
```

Comment:
Just a note. Even using 19.6, this argument need to remain 19.3 for compatibility with the volume created. There's a ticket open to improve how we get the wallets, and them this will not be needed.